src/argsparser.c

```diff
@@ -301,6 +302,20 @@ void parse_recurring_delay(char *arg)
     free(x);
 }
 
+int isnumber(char *pin)
```

Comment:
it's not my intention to annoy you, i just thought "if you do it, do it right". anyway, if i'm asking for more than you're willing to do, let's leave it at "isnumber" and i will fix it myself.